spec/helpers/registration_helper_spec.rb

```diff
@@ -33,6 +33,12 @@
     end
   end
 
+  describe '#valid_content_types' do
+    it 'returns the expected content types in the expected order' do
+      expect(valid_content_types).to eq ['Book (ltr)', 'Book (rtl)', 'File', 'Image', 'Map', 'Media', '3D', 'Document', 'Geo', 'Webarchive-seed']
+    end
+  end
```

Comment:
this verifies that our refactoring returns the expected list in the expected order